core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -330,6 +330,28 @@ private static String toBase64_(byte[] bytes) {
     }
   }
 
+  /** SQL BASE64(string) function for string. */
+  public static String base64(String string) {
+    return toBase64_(string.getBytes(UTF_8));
+  }
+
+  /** SQL BASE64(string) function for bytestring. */
+  public static String base64(ByteString byteString) {
+    byte[] bytes = byteString.getBytes();
+    return toBase64_(bytes);
+  }
+
+  /** SQL UNBASE64(string) function. */
+  public static @Nullable String unBase64(String base64) {
+    try {
+      base64 = FROM_BASE64_REGEXP.matcher(base64).replaceAll("");
+      //CHECKSTYLE: IGNORE 1
```

Comment:
What's this comment for?

Reply:
Calcite can not support new String method，use this comment to suppress it. Other place in Calcite also use it.